<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>569</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>P11Gui</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../assets/icons.qrc">
    <normaloff>:/icons/smartcard</normaloff>:/icons/smartcard</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0" colspan="2">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="2">
        <widget class="QToolButton" name="soLoginBtn">
         <property name="text">
          <string>SO Login</string>
         </property>
         <property name="icon">
          <iconset resource="../assets/icons.qrc">
           <normaloff>:/icons/lock</normaloff>:/icons/lock</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Reader</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cmbReader"/>
         </item>
        </layout>
       </item>
       <item row="0" column="1">
        <widget class="QToolButton" name="userLoginBtn">
         <property name="text">
          <string>User Login</string>
         </property>
         <property name="icon">
          <iconset resource="../assets/icons.qrc">
           <normaloff>:/icons/lock</normaloff>:/icons/lock</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>32</width>
           <height>32</height>
          </size>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <property name="toolButtonStyle">
          <enum>Qt::ToolButtonTextUnderIcon</enum>
         </property>
        </widget>
       </item>
       <item row="1" column="0" colspan="3">
        <widget class="QSplitter" name="splitter">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <widget class="QTableWidget" name="smartcardContent">
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <column>
           <property name="text">
            <string>Type</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Label</string>
           </property>
          </column>
         </widget>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Detail</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="0">
            <widget class="QToolButton" name="deleteBtn">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Delete</string>
             </property>
             <property name="icon">
              <iconset resource="../assets/icons.qrc">
               <normaloff>:/icons/trash</normaloff>:/icons/trash</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonTextBesideIcon</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="3">
            <widget class="QTextBrowser" name="detailText"/>
           </item>
           <item row="0" column="1">
            <widget class="QToolButton" name="saveBtn">
             <property name="text">
              <string>Save</string>
             </property>
             <property name="icon">
              <iconset resource="../assets/icons.qrc">
               <normaloff>:/icons/save</normaloff>:/icons/save</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>32</width>
               <height>32</height>
              </size>
             </property>
             <property name="toolButtonStyle">
              <enum>Qt::ToolButtonTextBesideIcon</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRefresh"/>
   <addaction name="actionchangePin"/>
   <addaction name="actionImport_p12"/>
   <addaction name="actiongenerateKeyPair"/>
   <addaction name="separator"/>
   <addaction name="actionSettings"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>775</width>
     <height>19</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <action name="actionImport_p12">
   <property name="icon">
    <iconset resource="../assets/icons.qrc">
     <normaloff>:/icons/import</normaloff>:/icons/import</iconset>
   </property>
   <property name="text">
    <string>Import p12</string>
   </property>
  </action>
  <action name="actionRefresh">
   <property name="icon">
    <iconset resource="../assets/icons.qrc">
     <normaloff>:/icons/refresh</normaloff>:/icons/refresh</iconset>
   </property>
   <property name="text">
    <string>Refresh</string>
   </property>
  </action>
  <action name="actiondeleteKey">
   <property name="text">
    <string>deleteKey</string>
   </property>
  </action>
  <action name="actionchangePin">
   <property name="icon">
    <iconset resource="../assets/icons.qrc">
     <normaloff>:/icons/change_pin</normaloff>:/icons/change_pin</iconset>
   </property>
   <property name="text">
    <string>changePin</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="../assets/icons.qrc">
     <normaloff>:/icons/settings</normaloff>:/icons/settings</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actiongenerateKeyPair">
   <property name="icon">
    <iconset resource="../assets/icons.qrc">
     <normaloff>:/icons/keypair</normaloff>:/icons/keypair</iconset>
   </property>
   <property name="text">
    <string>generateKeyPair</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../assets/icons.qrc"/>
 </resources>
 <connections/>
</ui>
